generatebundlefile/data/bundles_dev/1-26.yaml

```diff
@@ -0,0 +1,93 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-26-1001"
+kubernetesVersion: "1.26"
+packages:
+  - org: aws-containers
+    projects:
+      - name: hello-eks-anywhere
+        repository: hello-eks-anywhere
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+      - name: eks-anywhere-packages
+        repository: eks-anywhere-packages
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+      - name: eks-anywhere-packages-crds
+        repository: eks-anywhere-packages-crds
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+      - name: eks-anywhere-packages-migrations
+        repository: eks-anywhere-packages-migrations
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+  - org: aws-observability
+    projects:
+      - name: adot
+        repository: adot/charts/aws-otel-collector
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+  - org: cert-manager
+    projects:
+      - name: cert-manager
+        workloadonly: true
+        repository: cert-manager/cert-manager
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+          - name: latest
+  - org: harbor
+    projects:
+      - name: harbor
+        repository: harbor/harbor-helm
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: latest
+  - org: kubernetes
+    projects:
+      - name: cluster-autoscaler
+        repository: cluster-autoscaler/charts/cluster-autoscaler
+        registry: public.ecr.aws/l0g8r8j6
+        versions:
+            - name: 9.21.0-1.26-latest
```

Comment:
How do you map this version to the helm chart? I cannot find this version in 857 ECR

Reply:
That's a great question, I figured new builds were happening for 1.25 and 1.26 but I guess not? https://github.com/aws/eks-anywhere-build-tooling/tree/main/projects/kubernetes/autoscaler

Reply:
Ok it is in the repo, you might have been looking in the old chart repo.

Reply:
I was looking for the exact 9.21.0-1.26-latest tag, not knowing the presence of the regular expression look up.